requests-for-comments/2022-11-02-onboarding-2.0-continued.md

```diff
@@ -0,0 +1,123 @@
+# Request for comments: Onboarding 2.0 (continued) - Emanuele Capparelli / Raquel Smith
+
+## Problem statement
+
+With this effort we're tackling a couple different problems with onboarding:
+
+1. Current activation rates for organizations are lower than we'd like. This means that too many people sign up for PostHog, but then fail to get set up properly or see value before they get tired and leave.
+2. Current activation rates for session recordings (even after successful event ingestion) are lower than we'd like. This means that people get set up, but then don't use the session recording product.
```

Comment:
Fly by feedback - these two problems are really vaguely framed, and don't give helpful context. What does 'lower than we'd like' mean? Is there actually a problem?

For example, is it that the rate is X% but the industry standard is Y%? Or has the rate been consistently falling for a period of time? Or do we just have an OKR to increase our activation rate? You also don't define what activation rate is here. 

(I know I'm being pedantic - all of your proposed ideas sound worthwhile, it's just that the problem statement is weak and in future you could risk working on something that's actually fine at the expense of a more urgent problem.)

Reply:
It's just that we're on a public repo, and those are internal metrics, and we're trying to move RFCs as much as possible to a public repo. Should we move it out, so that we can discuss freely the objectives?

Reply:
Personally I think it's fine to default to transparency and share some information activation rate in a public RFC given we share these kinds of metrics in our OKRs publicly in the Handbook? (e.g. [weekly signup numbers](https://posthog.com/handbook/small-teams/marketing)). I don't think it hurts too much as every company defines their activation rate differently. 

(However it's your call - I understand why you wrote it like this now at least!)

Reply:
Transparency ftw then